<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>image &lt;-&gt; base64</string>
  </property>
  <property name="windowIcon">
   <iconset resource="appicon.qrc">
    <normaloff>:/img/ico/app.ico</normaloff>:/img/ico/app.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="group_image">
     <property name="title">
      <string>Image : Drag and drop an image file below</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="PhotoLabel" name="viewer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Panel</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hbox1">
        <item>
         <widget class="QPushButton" name="aboutButton">
          <property name="toolTip">
           <string>Show about dialog</string>
          </property>
          <property name="text">
           <string>&amp;About...</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hs1">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pasteButton">
          <property name="toolTip">
           <string>Paste image from clipboard</string>
          </property>
          <property name="text">
           <string>&amp;Paste</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openButton">
          <property name="toolTip">
           <string>Open and load image file</string>
          </property>
          <property name="text">
           <string>&amp;Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vbox">
     <item>
      <widget class="QToolButton" name="codeButton">
       <property name="toolTip">
        <string>Refresh codes from image</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="appicon.qrc">
         <normaloff>:/img/ico/update_codes.ico</normaloff>:/img/ico/update_codes.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="arrowType">
        <enum>Qt::ArrowType::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="imageButton">
       <property name="toolTip">
        <string>Refresh image from codes</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="appicon.qrc">
         <normaloff>:/img/ico/update_image.ico</normaloff>:/img/ico/update_image.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="arrowType">
        <enum>Qt::ArrowType::NoArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="group_codes">
     <property name="title">
      <string>Base64 : Paste codes below</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="lineWrapMode">
         <enum>QTextEdit::LineWrapMode::WidgetWidth</enum>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hbox2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Length : </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hs2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="toolTip">
           <string>Add Markdown head</string>
          </property>
          <property name="text">
           <string>&amp;Markdown</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="toolTip">
           <string>Choose image fotmat</string>
          </property>
          <item>
           <property name="text">
            <string>png</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>jpg</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ico</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bmp</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copyButton">
          <property name="toolTip">
           <string>Copy upper codes to clipboard</string>
          </property>
          <property name="text">
           <string>&amp;Copy</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="toolTip">
           <string>Save image or codes to local file</string>
          </property>
          <property name="text">
           <string>&amp;Save as...</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhotoLabel</class>
   <extends>QLabel</extends>
   <header>photolabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="appicon.qrc"/>
 </resources>
 <connections/>
</ui>
